### DISCUSSION TITLE
WiX 5 SummaryInformation 'Comments'

### DISCUSSION THREAD

**Discussion by mosBrkr** [wixtoolset/issues#8093] [Questions] [answered]
Hey guys

Sorry for the duplication, but as I didn't get a reply yet here (https://github.com/wixtoolset/issues/issues/7369#issuecomment-1993822037) I start another thread.

Even though I have added (or at least I think so) support for the 'Comments' attribute in the 'SummaryInformation' with my PR's I can't figure out how to test it using the new WiX 5 release candidates.

Quick steps for reproduction:
1. Create a new project (package)
2. Build the project -> Success
3. Add a 'SummaryInformation' with a 'Comments' attribute like
![image](https://github.com/wixtoolset/issues/assets/87169536/7ac9aced-6c15-4162-a8a0-97c024297d8c)
4. Build the project -> Build fails with error message
![image](https://github.com/wixtoolset/issues/assets/87169536/8120826f-4f14-4db7-a458-862f93912f10)

**Comment by ds-andreasor** [comment]
The https://wixtoolset.org/docs/schema/wxs/summaryinformation/ says that this attribute is v5 and newer only.
But right now you can't build v5 packages from Visual Studio so I guess it still uses v4 and the error is right.

**Reply by mosBrkr** [reply]
Is there a possibility where I can build a v5 package without Visual Studio?

**Reply by robmen** [reply]
Yes, you can always build without Visual Studio using MSBuild from the command-line. The steps are the same as I detailed (above or below, depending on GitHub sort order) about updating the `Sdk` attribute of your .wixproj.

**Comment by robmen** [comment, marked as answer]
You need to manually update your .wixproj to point to a pre-release of WiX. HeatWave will not do that for you. Fortunately, the steps are pretty simple:

1. Double-click on your project in the Solution Explorer
2. Edit the `Project` element's `Sdk` attribute from `WixToolset.Sdk/4.0.0` to `WixToolset.Sdk/5.0.0-rc.2`
3. Save
4. _Maybe_ have to refresh or close and re-open the solution
5. Rebuild your .wixproj

If you have more than one .wixproj in your Solution, I'm pretty sure you have to update all of them to the same version.

**Reply by mosBrkr** [reply]
Alright! Thanks for the explanations👍
